drake-torch.dockerfile

```diff
@@ -190,6 +198,12 @@ RUN cd $HOME && git clone https://github.com/lcm-proj/lcm \
     && cd lcm && mkdir -p build && cd build && cmake .. && make && make install \
     && cd $HOME && rm -rf lcm
 
+# install libfranka system-wide
+RUN cd $HOME && git clone https://github.com/frankaemika/libfranka.git \
+    && cd libfranka %% git checkout 0.5.0 && git submodule update --init \
```

Comment:
Should `cd libfranka %% git checkout 0.5.0` be `cd libfranka && git checkout 0.5.0` instead?

Reply:
Yes!

@dmsj Why are we installing libfranka system-wide? Isn't it an external in `drake-franka-driver`?

Reply:
It is no longer an external.